### DISCUSSION TITLE
Command Line Switches - Are there command line switches available? Would like to pass web page variables to command line download

### DISCUSSION THREAD

**Discussion by gnvdude** [LibreScore/dl-librescore#82] [Q&A] [answered]
This doesn't work, it still prompts you for the path. Looking for path, type (pdf vs mp3) and download local path output

npx dl-librescore https://musescore.com/asianmoofin/megalovania-clarinet-solo --output /musescore

![image](https://github.com/user-attachments/assets/1ae6908e-2b90-4466-8cfa-60629f732162)

**Comment by PeterNjeim** [comment, marked as answer]
Your syntax isn't correct. In order to input the URL, you need to write `--input` or `-i`
